sites/partners/src/users/JurisdictionAndListingSelection.tsx

```diff
@@ -0,0 +1,203 @@
+          <GridSection title={t("t.jurisdiction")} columns={4}>
+            <GridCell>
+              <Field
+                id="jurisdiction_all"
+                name="jurisdiction_all"
+                label={t("users.allJurisdictions")}
+                register={register}
+                type="checkbox"
+                inputProps={{
+                  onChange: (e: React.ChangeEvent<HTMLInputElement>) =>
+                    updateAllJurisdictionCheckboxes(e),
+                }}
+              />
+
+              <FieldGroup
+                name="jurisdictions"
+                fields={jurisdictionOptions}
+                type="checkbox"
+                register={register}
+                error={!!errors?.jurisdictions}
+                errorMessage={t("errors.requiredFieldError")}
+                validation={{ required: true }}
+                dataTestId={"jurisdictions"}
+              />
+            </GridCell>
+          </GridSection>
+          {selectedJurisdictions && (
+            <GridSection columns={4}>
+              {Object.keys(listingsOptions).map((key) => {
+                if (!selectedJurisdictions.includes(key)) {
+                  return null
+                }
+                const jurisdictionLabel = jurisdictionOptions.find((elem) => elem.id === key)?.label
+                return (
+                  <GridCell key={`listings_${key}`}>
+                    <GridSection
+                      title={t("users.jurisdictionalizedListings", {
+                        jurisdiction: jurisdictionLabel,
+                      })}
+                      columns={1}
+                    >
+                      <GridCell>
+                        <Field
+                          id={`listings_all_${key}`}
+                          name={`listings_all_${key}`}
+                          label={t("users.alljurisdictionalizedListings", {
+                            jurisdiction: jurisdictionLabel,
+                          })}
+                          register={register}
+                          type="checkbox"
+                          inputProps={{
+                            onChange: (e: React.ChangeEvent<HTMLInputElement>) =>
+                              updateAllCheckboxes(e, key),
+                          }}
+                        />
+
+                        <FieldGroup
+                          name="user_listings"
+                          fields={listingsOptions[key]}
+                          type="checkbox"
+                          register={register}
+                          error={!!errors?.user_listings}
+                          errorMessage={t("errors.requiredFieldError")}
+                          validation={{ required: true }}
+                          dataTestId={`listings_${jurisdictionLabel}`}
+                        />
+                      </GridCell>
+                    </GridSection>
+                  </GridCell>
+                )
+              })}
```

Comment:
This looks pretty similar to what is below starting on line 161. Can these use the same component?